### PR TITLE
dedupe pipe end textures

### PR DESCRIPTION
**Pull request by temp1011**
As the ends are opaque anyway is doesn't make any sense to have an opaque copy of them. This reduces the size of the texture map by ~5% I believe.

texture atlas before: before

texture atlas after: after

The size and placement of textures still needs a fair amount of work though.

### CODE REVIEW

**Comment by pupnewfster** [comment]
Looks nice (we still have to test it), but especially given how large the pipe textures are that does reduce it a fair bit.

**Comment by pupnewfster** [comment]
Actually thinking about it a bit more, and looking at it. Could we maybe just entirely remove the `#center_opaque` line? And then update the models to use the `#center` one in both cases?

**Comment by temp1011** [comment]
Could do, but then I would need to write another model file since there are some cables that have different opaque ends. I think the files are `.obj` too so I'm not sure what to do about that.

File: src/main/resources/assets/mekanism/blockstates/restrictive_transporter.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
       "#side": "mekanism:blocks/models/multipart/RestrictiveTransporterVertical",
       "#center": "mekanism:blocks/models/multipart/RestrictiveTransporter",
       "#side_opaque": "mekanism:blocks/models/multipart/opaque/RestrictiveTransporterVertical",
-      "#center_opaque": "mekanism:blocks/models/multipart/RestrictiveTransporter"
+      "#center_opaque": "mekanism:blocks/models/multipart/opaqueRestrictiveTransporter"
```
Comment (pupnewfster) [inline src/main/resources/assets/mekanism/blockstates/restrictive_transporter.json]: seems to be a missing slash

Comment (temp1011) [reply src/main/resources/assets/mekanism/blockstates/restrictive_transporter.json]: oops

**Pull request merged**